### ISSUE TITLE
Resolving a thumps down results in a server error

### ISSUE THREAD

**Issue by tordans** [mapbox/osmcha-frontend#394]
I am on https://osmcha.mapbox.com/changesets/74722824/?aoi=aa999729-e5f4-408a-914b-b7318f6b2b16

There was a thumbs down put there.

I see a secondary dropdown which allows to resolve the thumbs down.

Bug: Once I click one of the options I get an error:
> Error
> Authentication error. Sign in again and repeat the operation.

<img width="320" alt="Bildschirmfoto 2019-09-24 um 07 10 29" src="https://user-images.githubusercontent.com/111561/65482837-98b77300-de9a-11e9-8607-21f81e3be253.png">

**Comment by willemarcel** [comment]
@tordans was the changeset reviewed by you? Only the user that reviewed the changeset can add the tags, but I think we could review it and let at least the more experienced users add tags.

and of course the message error needs to be improved

**Comment by tordans** [comment]
> was the changeset reviewed by you?

No, by !bm. However, the UI suggest that I may solve it, since it is there and active (not grayed out or inactive). 

I think one could also argue, that any reviewer except for the reviewed person (the changeset-author) sould be able to solve the review. That would be a lot more open. Ideally with some kind of way for the reviewer to find out about issues that have been closed by someone else (email-notification, in-app-message, filter "show all that I reviewed bad but someone else closed as solved", etc.)

IMO, the first would be to fix the current feature.

Another good step afterward where to re-evaluate it, but IMO this is a lot more work, since the usecases are quite broad.
